### ISSUE TITLE
Does Adguard VPN sometimes reset my connection, or does it affect my overall internet connection?

### ISSUE THREAD

**Issue by dioarame** [AdguardTeam/AdGuardVPNExtension#154]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN Browser Extension version

2.0.65

### Environment

```markdown
- OS: Win11
- Browser: MS Edge
```

### Issue Details

I'm using Adguard VPN via subscription.
The tunneling feature, which allows the VPN to only work on certain sites or program, doesn't work well on the desktop program, so I'm using it on Edge with the extension.

However, I am still experiencing anomalies and would like to contact you.
Typically, the site I keep tunneling on is Twitch. It works as expected, but the connection seems to drop intermittently.
The problem is, it doesn't just disconnect the tab in question, it disconnects the entire internet connection.

For example, if I'm playing a game, I get a message saying there's an error connecting to the server and I need to reconnect, or if I'm downloading, I get a 0KB speed reading.

Of course, we've found that after a short period of time - about 10 seconds - the connection works fine, but in most cases, even if it does, you'll need to "refresh" to reflect that.

I don't have a lot of knowledge about VPNs, but I recognize the situation as intermittent disconnects or resets.

I'm hoping to get an answer as to whether this is due to something I'm not doing, or if it's a technical glitch.

Below are captures of the only settings I have in the extension, noting that everything else is by default.

### Screenshots

![image](https://github.com/AdguardTeam/AdGuardVPNExtension/assets/40052197/cac87a46-cf55-406e-a5a1-89f509f13f99)

**Comment by Birbber** [comment]
@dioarame Hello! Thank you for the report.
We've recently released [version 2.1.5](https://github.com/AdguardTeam/AdGuardVPNExtension/releases/tag/v2.1.5). Can you please check if the problem persists?

**Comment by Birbber** [comment]
@dioarame Any news?

**Comment by dioarame** [comment]
After checking, I realized that this was more of an Adguard issue than a problem with AdguardVPN.
In general, when I logged the number of times I was disconnected, it was overwhelmingly when an ad appeared in the stream, so I adjusted the settings to account for that.
Currently, I'm not experiencing this issue, but that's because I turned off the DNS protection option in Adguard's own settings. 
* Changing DNS filtering rules and DNS protocols did not make a difference to the internet disconnections.

Anyway, I haven't experienced any internet disconnections at this point, but turning off DNS protection is a big change.
